src/trackedit/internal/trackeditinteraction.h

```diff
@@ -71,6 +71,11 @@ class TrackeditInteraction : public ITrackeditInteraction, public muse::Injectab
 
     bool toggleStretchToMatchProjectTempo(const ClipKey& clipKey) override;
 
+    int64_t clipGroupId(const trackedit::ClipKey& clipKey) const override;
+    void setClipGroupId(const trackedit::ClipKey& clipKey, int64_t id) override;
+    int64_t determineGroupId(const ClipKeyList& clipKeyList) const override;
+    ClipKeyList clipsGroup(int64_t id) const override;
```

Comment:
Can we name it `clipsInGroup`? Current name makes me think that it should return a group id or something.